libmattermost.c

```diff
@@ -1196,6 +1217,40 @@ mm_get_users_by_ids(MattermostAccount *ma, GList *ids)
 	g_free(url);       
 }
 
+#define _MM_TOOLTIP_LINE_ADD(b,u,d,p,o) \
+{ \
+	if (o) { \
+		purple_notify_user_info_add_pair(u,d,o); \

```

Comment:
This part didn't work with purple-3 as the `purple_notify_user_info_add_pair` function no longer exists